contracts/scripts/native_solc_compile_all_ccip

```diff
@@ -8,9 +8,7 @@ echo " └───────────────────────
 
 SOLC_VERSION="0.8.24"
 OPTIMIZE_RUNS=26000
```

Comment:
q: does `FeeQuoter` still fit into 26k runs?

Reply:
This should be tested here I believe https://github.com/smartcontractkit/chainlink/blob/48982652f992bdf6498d1ab8d52ba5abd35727ef/core/gethwrappers/ccip/deployment_test/deployment_test.go#L72